modules/docker-mirror/variables.tf

```diff
@@ -81,3 +81,9 @@ variable "randomize_resource_names" {
   type        = bool
   description = "Use randomized names for resources. Disable if you are upgrading existing executors that were deployed using the legacy naming conventions, unless you want to recreate executor resources on GCP."
 }
+
+variable "use_local_ssd" {
+  type        = bool
+  default     = false
+  description = "Use a local SSD for the data dir of the registry instead of a persistent disk. This will mean that the cache will reset after the instance is replaced! disk_size is also not honored when true."
```

Comment:
Given this description, it's not really clear what the benefits of the local SSD are. It wouldn't really prompt me to enable it as an end user which would mean I would miss out on the big gains

Reply:
yeah I'm not sure if I totally want end-users to use this just yet - it's very finicky to document when to use it 🙈 
- If you don't care about cost, do it
- If you provision at least ~100GB of pd-ssd disk, do it
- But only if you use an instance type that supports local SSDs and also only (for now) if that instance type supports having a single one (>8 vCPU for example on the C2 platform requires 2, and so on) - which would require that we support multiple disks and mounting them in a RAID0 which the startup script doesn't yet support. 

WDYT if we let this run for a bit longer on dotcom auto indexing, then see if we can get this to MIs and make sure it works well there, too and then invest the additional ~1d to make it work properly for larger machines, too and then document when to use it?

Reply:
👍  makes sense to me